src/pages/FolderPage.jsx

```diff
@@ -0,0 +1,89 @@
+import { useState } from 'react';
+import { styled } from 'styled-components';
+
+import useUserFoldersQuery from 'hooks/api/folder/useUserFoldersQuery';
+
+import PageTitle from 'components/common/PageTitle';
+import FolderHeader from 'components/folder/FolderHeader';
+import MainLayout from 'components/template/MainLayout';
+import AddFolderButton from 'components/common/button/AddFolderButton';
+import TabButton from 'components/common/button/TabButton';
+import OptionBtns from 'components/folder/OptionBtns';
+import FolderGridCard from 'components/folder/FolderGridCard';
+import FloatButton from 'components/common/button/FloatButton';
+
+const Styled = {
+  NoLink: styled.div`
+    height: 10rem;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  `,
+
+  ButtonBox: styled.div`
+    margin-bottom: 2.4rem;
+
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  `,
+
+  TitleAndOptions: styled.div`
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    margin-bottom: 2.4rem;
+
+    @media (max-width: 767px) {
+      flex-direction: column;
+      align-items: start;
+      gap: 1.2rem;
+    }
+  `,
+};
+
+function FolderPage() {
+  const { data } = useUserFoldersQuery({ userId: 1 });
```

Comment:
현재 000API폴더 내 api요청 함수들은 그냥 요청하고 리턴하는 형식으로 했는데 이걸 쿼리에서 받아서 { data }로 할당받아 넘겨주는 식으로 해야하는지 useQuery를 사용하는곳에서 data로 받도록 하는 식이 맞는지 궁금합니다.

-> 넵 이런 식으로 많이 씁니다! 객체 구조 분해 할당을 통해서 해당 API가 로딩 상태인지 확인하는 isLoading을 통해서 로딩인 것을 확인하고 Loading Progress를 보여준다던지 할 수 있어요.
만약 다른 쿼리도 있는 페이지라면, 

```suggestion
  const { data: foldersData } = useUserFoldersQuery({ userId: 1 });
```
와 같이 써볼 수도 있겠네요!